src/models/account.model.ts

```diff
@@ -0,0 +1,41 @@
+import Setting from './setting.model';
+import {
+  Column,
+  Model,
+  Table,
+  DeletedAt,
+  CreatedAt,
+  UpdatedAt,
+  HasMany,
+  AllowNull,
+  Unique,
+  PrimaryKey,
+  DataType,
+} from 'sequelize-typescript';
+
+@Table
```

Comment:
We need to add table names and list down the indexes on the able if any
for example -
```
@Table({
  tableName: ,
  indexes: [
    {
      name: 'name_of_index',
      fields:   ['field_1', 'field_2'],
      unique: boolean
    }
  ]
})
```
Make the same change for the settings model as well

Reply:
For the accounts model will the `@Unique` decorator suffice for one field? Instead of defining at the table level.